dependencies.props

```diff
@@ -41,7 +41,7 @@
     <AppXRunnerVersion>1.0.3-prerelease-00921-01</AppXRunnerVersion>
     <XunitPerfAnalysisPackageVersion>1.0.0-beta-build0007</XunitPerfAnalysisPackageVersion>
     <TraceEventPackageVersion>1.0.3-alpha-experimental</TraceEventPackageVersion>
-    <XunitNetcoreExtensionsVersion>1.0.1-prerelease-01911-02</XunitNetcoreExtensionsVersion>
+    <XunitNetcoreExtensionsVersion>1.0.1-prerelease-02116-01</XunitNetcoreExtensionsVersion>
```

Comment:
Would it be possible to include this in the auto-updates?  

Basically, just need to add 

```xml
     <XmlUpdateStep Include="BuildTools">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>XunitNetcoreExtensionsVersion</ElementName>
       <PackageId>Microsoft.xunit.netcore.extensions</PackageId>
     </XmlUpdateStep>
```

on line 150, under the current UpdateSteps.